pkg/collector/corechecks/containers/docker.go

```diff
@@ -341,6 +341,27 @@ func (d *DockerCheck) Run() error {
 	return nil
 }
 
+func (d *DockerCheck) reportIOMetrics(io *cmetrics.CgroupIOStat, tags []string, sender aggregator.Sender) {
+	if io == nil {
+		return
+	}
+	// Read values, per device or fallback to sum
+	for dev, value := range io.DeviceReadBytes {
```

Comment:
should this be placed in the `else` of the condition bellow?

Reply:
yes, that will make the intention more clear